src/client/app/graph/arrow/arrow.component.ts

```diff
@@ -78,4 +89,70 @@ export class ArrowComponent implements OnInit {
     return pointsStr;
   }
 
+  onMousedown(event: MouseEvent) {
+    if (EventUtils.eventIsFromPrimaryButton(event)) {
+      this.tipDragging = true;
```

Comment:
Just the tip?  😄 

Seems like the entire arrow is actually draggable? While we're on the subject, consider adding a TODO to invisibly increase the hitbox for an arrow?

Reply:
Smh. It's either "tip" or "head", both of which could mean other things. Lol
You're right, the mousedown listener should just be on the tip. Added a TODO.